apps/docs/content/guides/platform/access-control.mdx

```diff
@@ -1,85 +1,270 @@
+|                                  | Revoke                 | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |           <IconX size={14} />           |                       <IconX size={14} />                       |
+|                                  | List                   | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |                       <IconX size={14} />                       |
+| **Branching**                    |                        |                                         |                                         |                                         |                                                                 |
+| Enable branching                 | -                      | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |           <IconX size={14} />           |                       <IconX size={14} />                       |
+| Disable branching                | -                      | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |           <IconX size={14} />           |                       <IconX size={14} />                       |
+|                                  | Create                 | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |                       <IconX size={14} />                       |
+|                                  | Delete                 | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |                       <IconX size={14} />                       |
+|                                  | List                   | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> | <IconCheck size={14} color="#3FCF8E" /> |             <IconCheck size={14} color="#3FCF8E" />             |
+
+[^1]: Available on the Team and Enterprise Plans.
+[^2]:
     Invites sent from a SSO account can only be accepted by another SSO account
     coming from the same identity provider. This is a security measure that
     prevents accidental invites to accounts not managed by your company's
     enterprise systems.
 
-[^2]: Available on the Teams and Enterprise Plans.
-[^3]:
-    Only available on projects using PostgreSQL 14 and above. You can upgrade
-    your project through [infrastructure settings](https://supabase.com/dashboard/project/_/settings/infrastructure).
+[^3]: Available on the Team and Enterprise Plans.
+[^4]: Only available on projects using PostgreSQL 14 and above. You can upgrade your project through [infrastructure settings](https://supabase.com/dashboard/project/_/settings/infrastructure).
```

Comment:
This limitation can be mentioned on just the read replica page. needn't be on the permissions page.